futures-cpupool/src/lib.rs

```diff
@@ -252,3 +268,61 @@ impl<F: Future> Future for Sender<F, Result<F::Item, F::Error>> {
         Ok(Async::Ready(()))
     }
 }
+
+impl Builder {
+    /// Create a builder a number of workers equal to the number
+    /// of CPUs on the host.
+    pub fn new() -> Builder {
+        Builder {
+            pool_size: num_cpus::get(),
+            after_start: None,
+            before_stop: None,
+        }
+    }
+
+    /// Set size of a future CpuPool
```

Comment:
Perhaps this could mention that "size" is the number of threads that will be spawned?